src/pages/Manual/Manual.tsx

```diff
@@ -1,9 +1,90 @@
-import { FC } from 'react'
+import { map } from 'lodash'
+import { FC, useCallback, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import BaseButton from 'components/atoms/BaseButton/BaseButton'
+import useHashState from 'hooks/useHashState'
+import ReactHtmlParser from 'html-react-parser'
+import helpSections from './helpSections.json'
+import { HelpSections } from 'components/organisms/Tooltip/Tooltip'
 
-// TODO: WIP - implement this page
+import classes from './classes.module.scss'
 
 const Manual: FC = () => {
-  return <h1>Manual</h1>
+  const { t } = useTranslation()
+
+  const helpSectionsData: HelpSections = helpSections
+
+  const date = '15.02.2022'
+  const author = 'Interlex'
+  const authorEmail = 'koostaja@interlex.ee'
+
+  const { setHash, currentHash } = useHashState()
+
+  const handleTitleClick = useCallback(
+    (key: string) => {
+      setHash(key)
+      scrollToElement(key)
+    },
+    [setHash]
+  )
+
+  const scrollToElement = (key: string) => {
+    const matchingElement = document.getElementById(key)
+    if (matchingElement) {
+      matchingElement.scrollIntoView({
+        behavior: 'smooth',
+        block: 'start',
+      })
+    }
+  }
+
+  useEffect(() => {
+    scrollToElement(currentHash)
+  }, [currentHash])
+
+  const manualTitle = map(helpSectionsData, ({ title }, key) => {
+    return (
+      <BaseButton
+        key={key}
+        onClick={() => handleTitleClick(key)}
+        className={
+          currentHash === key ? classes.activeTitle : classes.manualTitle
+        }
+      >
+        {title || ''}
+      </BaseButton>
+    )
+  })
+
+  const manualContent = map(helpSectionsData, ({ title, content }, key) => {
+    return (
+      <div key={key} id={key} className={classes.sectionContent}>
+        <h2>{title || ''}</h2>
+        <div className={classes.section}>{ReactHtmlParser(content) || ''}</div>
+      </div>
+    )
+  })
+
+  return (
+    <div className={classes.manualContainer}>
+      <h1 className={classes.title}>{t('menu.manual')}</h1>
+      <div className={classes.manualContentContainer}>
+        <div className={classes.articleInfoContainer}>
+          <p className={classes.articleInfo}>
+            {t('menu.article_created_by', { author, authorEmail })}
+          </p>
+          <p className={classes.articleInfo}>
+            {t('menu.last_changed', { date })}
+          </p>
+        </div>
+        {manualContent}
+      </div>
+      <div className={classes.manualTitleContainer}>
```

Comment:
**[blocker]**

The table of contents should always be visible and should be separately scrollable (Similar to how our SideBar works).

Reply:
**[blocker]**

It currently seems that part of the content in the table of content will be hidden. Added some extra items and currently the bottom 2 will be out of screen and can't be scrolled to.

Reply:
Missed that, fixed